### ISSUE TITLE
App crash when get Purchases or Buy item. (iOS)

### ISSUE THREAD

**Issue by Wpdas** [myflashlab/inAppPayments-ANE#27]
Application simply closes when trying to access any method after loading items.

**Comment by Wpdas** [comment]
I will try it https://github.com/myflashlab/inAppPayments-ANE/issues/26

**Comment by Wpdas** [comment]
(Solution)

My review: This problem occurred after the app did not go through review. I had to enter the items and change anything (in my case I just put or took a "." In the description). Then the status of the item returned as "Waiting for review." After that everything returned to normal. The app returned to get the item data.

**Comment by myflashlab** [comment]
could you please explain what do you mean by putting/taking a "."? a little more information can be very helpful for other devs who might face a similar problem. Thanks.

**Comment by Wpdas** [comment]
Of course!

I meant that I just went into the item description and changed a metadata field (description in the case) so that the item's status returned to "Waiting for review".

The problem is this: When you submit an item to be approved and it is disapproved for a crash, this item has status "Waiting for developer action". When the item has this status, the item is inaccessible.

To solve I did what I said above, changed some metadata and saved. Then returned the status to "Waiting for review". And the app again received the data.
